test/suites/device-usb-camera/main_test.go

```diff
@@ -8,10 +8,14 @@ import (
 )
 
 const (
-	deviceUSBCamSnap        = "edgex-device-usb-camera"
-	deviceUSBCamApp         = "device-usb-camera"
-	deviceUSBCamServicePort = "59983"
-	rtspServerPort          = "8554"
+	deviceUSBCamSnap    = "edgex-device-usb-camera"
+	deviceUSBCamApp     = "device-usb-camera"
+	deviceUSBCamRtspApp = "device-usb-camera/rtsp"
+)
+
+var (
+	deviceUSBCamPort     = utils.ServicePort(deviceUSBCamApp)
+	deviceUSBCamRtspPort = utils.ServicePort(deviceUSBCamRtspApp)
```

Comment:
I like this style because it requires the least changes to the code. Even though you ended up renaming `deviceUSBCamServicePort` to `deviceUSBCamPort`. I didn't know the compiler allows function calls outside of a function.

Reply:
right, the compiler only permits function calls for package level variables and constants outside of a function.